styles/styles.css

```
@@ -0,0 +1,611 @@
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 16px;
+    padding: 8px 20px;
+    width: 160px;
+    text-decoration: none;
+    background-color: #756157;
+    color: #FFFFFF;
+    text-transform: uppercase;
+}
+
+.reservations-button {
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 36px;
+    text-align: center;
+    width: 376px;
+    padding: 8px 50px;
+    background-color: #756157;
+    color:#FFFFFF;
+    text-decoration: none;
+}
+
+.about-hotel {
+    text-decoration: none;
+    text-align: center;
+    padding: 8px 50px;
+    width: 140px;
+    background-color: #756157;
+    color: #FFFFFF;
+}
+
+.add-to-favourites,
+.added-to-favourites {
+    text-decoration: none;
+    text-align: center;
+    padding: 8px 50px;
+    width: 140px;
+    background-color: #82B3D3;
+    color: #FFFFFF;
+}
+
+.added-to-favourites {
+    background-color: #7DB54F;
+
+}
+
+.button {
+    font-family: inherit;
+    padding: 8px 50px;
+    background-color: #82B3D3;
+    color:#FFFFFF;
+    font-size: 20px;
+}
+
+
+.welcome {
+    background-image: url(../images/index-background.jpg);
+    background-repeat: repeat;
+    background-size: cover;
+    background-position: top center;
+}
+
+.welcome-text {
+    display: block;
+    margin-left: auto;
+    margin-right: auto
+}
+
+
+
+.advantages-service-list {
+    display: flex;
+}
+
+
+.advantages-item-first,
+.advantages-item-fifth {
+    background-color: #82B3D3;
+    color:#FFFFFF;
+}
+
+.advantages-item-second,
+.advantages-item-fourth {
+    background-color: rgba(131, 179, 211, 0.12);
+}
+
+.advantages-item-third {
+    background-color: rgba(131, 179, 211, 0.2);
+}
+
+.advantages-service-item {
+    width: 400px;
+}
+
+.advantages-service-item-housing,
+.advantages-service-item-souvenirs {
+    background-color: rgba(131, 179, 211, 0.12);
+    color:#FFFFFF;
+}
+
+.advantages-service-item-housing {
+    background-image: url(../images/house.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+}
+
+.advantages-service-item-food {
+    background-image: url(../images/food.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+}
+
+.advantages-service-item-souvenirs {
+    background-image: url(../images/present.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+}
+
+.newsletter-background {
+    background-image: url("../images/newsletter-background.jpg");
+    color: #FFFFFF;
+    background-repeat: repeat;
+    background-size: cover;
+    background-position: center;
+    background-color: #82B3D3;
+}
+
+.newsletter,
+.newsletter-title {
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 36px;
+    text-transform: uppercase;
+    color: #FFFFFF;
+}
+
+
+
+
+.contacts-phone {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 40px;
+    color: #000000;
+    text-decoration: none;
+    text-align: center;
+}
```

```suggestion
.contacts-phone {
    font-weight: 400;
    font-size: 40px;
    line-height: 40px;
    color: #000000;
    text-decoration: none;
    text-align: center;
}
```

Я буду плакать от них